ADOPTERS.md

```diff
@@ -38,6 +42,14 @@ Internal Pods can get the real source IP of the request.
 
 You can contact us with mkt@transwarp.io
 
+**[TeraSky](https://terasky.com)**
+TeraSky is a Global Advanced Technology Solutions Provider.  
```

Comment:
You need to remove the tailing spaces after each line for them to form a single paragraph.

Reply:
done